### PR TITLE
Test PR

### PR DESCRIPTION
**Pull request by ashughes**

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -37,7 +37,7 @@
     <ConfirmationsSetting value="0" id="Add" />
     <ConfirmationsSetting value="0" id="Remove" />
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" assert-keyword="true" jdk-15="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" default="true" assert-keyword="true" jdk-15="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
Comment (ashughes) [inline .idea/misc.xml]:
```suggestion
  <component name="ProjectRootManager" version="3" languageLevel="JDK_1_7" default="true" assert-keyword="true" jdk-15="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```

File: app/build.gradle [hunk 2]
```diff
@@ -23,4 +23,7 @@ dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     testCompile 'junit:junit:4.12'
     compile 'com.android.support:appcompat-v7:23.3.0'
+    compile 'com.android.support:recyclerview-v7:23.3.0'
+
+    compile 'com.github.bumptech.glide:glide:3.7.0'
```
Comment (ashughes) [inline app/build.gradle]:
```suggestion
    compile 'com.github.bumptech.glide:glide:3.9.0'
```

File: app/src/main/java/edu/calpoly/android/recyclerviewdemo/AnimalUtils.java [hunk 3]
```diff
@@ -20,6 +24,41 @@ public String getName() {
...
             return mImageUrl;
         }
+
+        public boolean isDangerous() {
+            return mDangerous;
+        }
+
+        public void setDangerous(boolean dangerous) {
+            this.mDangerous = dangerous;
+        }
```
Comment (ashughes) [inline app/src/main/java/edu/calpoly/android/recyclerviewdemo/AnimalUtils.java]: This is not needed.

```suggestion
```

**Review by ashughes — COMMENTED** [review]
Test

File: app/src/main/java/edu/calpoly/android/recyclerviewdemo/AnimalUtils.java [hunk 4]
```diff
@@ -20,6 +24,38 @@ public String getName() {
...
             return mImageUrl;
         }
+
+        public boolean isDangerous() {
+            return mDangerous;
+        }
+
+
+        public int describeContents() {
+            return 0;
```
Comment (ashughes) [inline app/src/main/java/edu/calpoly/android/recyclerviewdemo/AnimalUtils.java:34]:
```suggestion
            return 1;
```

Comment (alenz316) [reply app/src/main/java/edu/calpoly/android/recyclerviewdemo/AnimalUtils.java:34]:
```suggestion
            return 2;
```

Comment (ashughes) [reply app/src/main/java/edu/calpoly/android/recyclerviewdemo/AnimalUtils.java:34]: Actually, I don't think we should change this after all.

**Pull request closed without merging**